atlp-rwanda/space-bn-frontend
# #175743390 ft(languages): support multiple languages

#### What does this PR do?

- Support multiple languages

#### Description of Task to be completed? 

- Install and set up `i18next`
- Install `i18next-http-backend` to fetch translations
- Add dropdown to change language
- Add languages `[English, French, Kinyarwanda, Swahili]` to be supported

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch **`ft-support-multiple-languages-#175743390`**

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start the server

- In your default browser navigate to **`http://localhost:3000`** 

> On the select choose any supported language and you should see text translated in the language of choice

#### Any background context you want to provide?

- N/A

#### What are the relevant Pivotal Tracker stories?

- [#175743390](https://www.pivotaltracker.com/n/projects/2476677/stories/175743390)

#### Screenshots (if appropriate)

- 
<img width="1440" alt="Screen Shot 2021-02-15 at 10 57 38" src="https://user-images.githubusercontent.com/54619678/107925558-468a5c80-6f7d-11eb-9bb3-52eb097c1841.png">


#### Questions:

- N/A
